### ISSUE TITLE
Backup Stickers or Store Stickers in iCloud

### ISSUE THREAD

**Issue by brygui** [alcor/stickerclip#5]
App is great. Having them stored in iCloud or having an export feature would be a great addition.

**Comment by icpryde** [comment]
Yes I second this but it would also be nice if we could delete stickers in the drawer or if we could maybe make a favourites section where the ones we make can live?
